src/main/java/org/jabref/model/entry/BiblatexEntryTypes.java

```diff
@@ -372,7 +372,7 @@ public String getName() {
                     FieldName.orFields(FieldName.YEAR, FieldName.DATE));
             addAllOptional(FieldName.SUBTITLE, FieldName.TITLEADDON, FieldName.LANGUAGE, FieldName.HOWPUBLISHED,
                     FieldName.TYPE, FieldName.VERSION, FieldName.NOTE, FieldName.ORGANIZATION, FieldName.LOCATION,
-                    FieldName.MONTH, FieldName.ADDENDUM, FieldName.PUBSTATE, FieldName.DOI, FieldName.EPRINT,
+                    FieldName.DATE, FieldName.MONTH, FieldName.YEAR, FieldName.ADDENDUM, FieldName.PUBSTATE, FieldName.DOI, FieldName.EPRINT,
```

Comment:
Why do you add Date and Year here? They fields are already in the required tab?

Reply:
See biblatex manual, 2.1.1 misc entry type. 
It's listed both in required and optional fields. Either it's an error in the spec or there is some reason why this is double?

>Required felds: author/editor, title, year/date
>Optional felds: subtitle, titleaddon, language, howpublished,
>type, version, note, organization, location, date, month, year,
>addendum, pubstate, doi, eprint, eprintclass, eprinttype, url,
>urldate

Reply:
Yes, that looks to me like an error in manual.